unity-client/Assets/Resources/Prefabs/SceneController.prefab

```diff
@@ -733,6 +734,49 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 9cb6cc2810479c5489bd7b3152757df6, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
+--- !u!1 &4412280342020653818
+GameObject:
```

Comment:
As all our "bridges" are under the `SceneController` gameObject, it wouldn't make more sense to have this added to it instead of a child GO?

Reply:
You are right, I just moved the monobehaviour inside SceneController and deleted the child GO (updated call in Kernel to point to the SceneController GO now).

The only concern I have with this, in the future we can analyze it, is that it's probably more performant to have these SendMessage calls pointing to GOs with just 1 specific monobehaviour, to avoid Unity's `SendMessage` call on every monobehaviour in the SceneController GO. But we are also migrating several messages to native messages, so we will probably reduce the overall usage of SendMessage soon.